vllm/hpu/attn_bias.py

```diff
@@ -0,0 +1,764 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
```

Comment:
Overall project is licensed under Apache. The LICENSE file contains neither the BSD license nor a Meta attribution.